### ISSUE TITLE
Move local helm chart to the official chart repository

### ISSUE THREAD

**Issue by Rowern** [hjacobs/kube-downscaler#29]
Would you accept that I start a PR on https://github.com/helm/charts to move the chart from your local repository to the official one ?

This would be more convenient for our deploy flow and also give more visibility to your project.

**Comment by hjacobs** [comment]
Yes, of course, go ahead! I would be very glad if you take care of this :smile:  I'm not using Helm myself, that's why I refrain from taking over Helm Chart maintenance.

**Comment by sakomws** [comment]
Great initiative! Feel free Rowern to ask any help.
